server/chat-commands/moderation.ts

```diff
@@ -286,9 +286,8 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
-			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
-		}
+		const showTrusted = group === ' ' && Users.isTrusted(targetId);
+		if (group !== ' ' || showTrusted) buffer.push(`Global auth: ${showTrusted ? 'trusted' : group}`);
```

Comment:
```suggestion
```suggestion
		if (group !== ' ' || Users.isTrusted(targetId)) {
			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
		}
``````